tests/cc46/output.ref

```diff
@@ -1590,7 +1593,7 @@ Doing transition
 
 Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]
 
-Properties computed using the CC2 density matrix
+Properties computed using the CC2 {} density matrix
```

Comment:
This doesn't look correct with the remaining `{}` or am I missing something?

Reply:
It was not, good catch!